### DISCUSSION TITLE
Student Processor - Gizmo

### DISCUSSION THREAD

**Discussion by PunkinMac** [gizmo-platform/gizmo#29] [Q&A]
When plugging into a computer, the student processor isn't turning on. The LED light is not coming on at all causing me to think the board is fried. I don't know if BEST can replace it, has anyone encountered this issue, and if so what's the solution?

**Comment by the-maldridge** [comment]
Jumping straight to "the board is fried" is a little pessimistic even for me :grin:.  What have you tried so far, and does firmware load mode work (Hold the bootsel button while plugging in the USB cable).  The LED on the board is user controlled, so unless you have code that controls that LED, it won't do anything.
